src/m365/flow/commands/run/run-list.ts

```diff
@@ -124,6 +130,10 @@ class FlowRunListCommand extends PowerAutomateCommand {
     try {
       const items = await odata.getAllItems<PowerAutomateFlowRun>(url);
 
+      if (args.options.output === 'json' && args.options.withTrigger) {
```

Comment:
I wonder if shouldn't we add a validation check for that?
if this new option only makes sense with `json` output the maybe we should already provide this feedback on validation to the user 🤔
@pnp/cli-for-microsoft-365-maintainers any comment on that?

Reply:
Makes sense to me.